pkg/testing/envtest_setup.go

```diff
@@ -43,6 +44,7 @@ func SetupEnvTest() *envtest.Environment {
 			filepath.Join("..", "..", "..", "config", "crd"),
 			filepath.Join("..", "..", "..", "test", "crds"),
 		},
+		UseExistingCluster: &useExistingCluster,
 	}
```

Comment:
You can use proto.bool for quick primitive booleans

Reply:
oh nice! I have been looking for a quick way to do for long time!